### DISCUSSION TITLE
Problem with  generating videos

### DISCUSSION THREAD

**Discussion by owlvman** [Comfy-Org/ComfyUI-Manager#384] [Q&A]
Hi, when generating videos, my images look very strange, does anyone know what the problem might be?
![зображення_2024-02-01_184848200](https://github.com/ltdrdata/ComfyUI-Manager/assets/81329176/22841662-fb0a-4423-b3e1-7c0a3cc23623)

**Comment by ltdrdata** [comment]
Here is the ComfyUI-Manager repository. The issue you are referring to should be addressed to the AnimateDiff extension repository. Just to clarify one thing, if you are using the _SDE sampler in KSampler, it should never be used with the normal scheduler. It is typically used with karras instead.

And in AnimateDiff, it is recommended to use euler/normal for the sampling/scheduler method.

**Reply by owlvman** [reply]
It`s work, thank you)
